apps/web/src/components/core/GlobalBanner/GlobalBanner.tsx

```diff
@@ -27,9 +31,10 @@ export function GlobalBanner({
   onClose,
   onClick,
   experienceFlag,
+  bannerVariant = 'default',
```

Comment:
turns out after previous refactoring passes, we _only_ had a hyperblue variant of the banner! reintroduced the default one